src/sonic-config-engine/portconfig.py

```diff
@@ -258,6 +259,14 @@ def __init__(self, name, bmode, properties):
 
         if not self._breakout_capabilities:
             raise RuntimeError("Unsupported breakout mode {}!".format(bmode))
+    def _parse_name_to_prefix_and_id(self):
```

Comment:
>_parse_name_to_prefix_and_id [](http://example.com/codeflow?start=8&length=28)

Leave 1~2 lines before and after this function. Not use 0 or too many empty lines.

Reply:
done